### PR TITLE
major cleanup (spam), deprecated speed_aggregate

### PR DESCRIPTION
**Pull request by pascal-sauer**

### CODE REVIEW

File: R/speed_aggregate.R [hunk 1]
```diff
@@ -233,7 +233,7 @@ speed_aggregate <- function(x,rel,fname=NULL,weight=NULL,from=NULL,to=NULL,dim=1
     if(length(grep("\\.csv$",rel))>0) {
       rel <- getAggregationMatrix(rel,from=from,to=to) 
     } else {
-      rel <- read.spam(rel)
+      stop("cannot read spam files with this luscale version, please use luscale <= 2.27.12")
```
Comment (tscheypidi) [inline R/speed_aggregate.R]:
```suggestion
      stop("cannot read spam files with this luscale version, please use luscale < 3.0.0")
```

**Review by tscheypidi — APPROVED** [review]
looks good, just one minor suggestion

File: R/speed_aggregate.R (hunk 1 above)
Comment (pascal-sauer) [reply R/speed_aggregate.R]: Funny, I had the exact same thing before, and then decided to switch it, so you have a concrete version that you can install directly. Do you still want to change it? I don't mind

Comment (tscheypidi) [reply R/speed_aggregate.R]: also not a strong preference. I thought it is more intuitive to clarify via the error message that the major version jump is also the cutoff point

**Pull request merged**